spec/tests/ContentMetadata.spec.ts

```diff
@@ -1,437 +0,0 @@
-import { PlayerType, StreamType, VRContentType } from 'bitmovin-player';
```

Comment:
I renamed this file into `ConvivaAnalytics`, but git tracks it as deleted for some reason.